app/views/documents/form_partials/_tags.html.erb

```diff
@@ -15,57 +15,39 @@
       <% end %>
     <% end %>
```

Comment:
Are there likely to be multiple errors here? Either way we can just loop over an empty array without the `any?` check. If there are possibly multiple errors should the wrapper tag be a `<div>` instead of a `<span>` so that they're not running together.

``` erb
<% form.object.errors[:tags].each do |error| %>
  <span id="status-error" class="govuk-error-message">
    <span class="govuk-visually-hidden">Error:</span>
    <%= error %>
  </span>
<% end %>
```